packages/compiler-cli/src/ngcc/src/packages/dependency_resolver.ts

```
@@ -28,7 +28,7 @@ import {EntryPoint} from './entry_point';
  */
 export interface InvalidEntryPoint {
   entryPoint: EntryPoint;
-  invalidDependencies: string[];
+  missingDependencies: string[];
```

By reading the comments, `missingDependencies` soundd more accurate.
But having looked at the code now, I realize there can actually be both missing and invalid dependencies (see https://github.com/angular/angular/pull/25862#discussion_r217412162).

I am not sure they should be treated the same though 😕 
Maybe invalid ones should throw an error (to err (literally) on the side of caution :grin:)